### ISSUE TITLE
Feature Request: Allow entered text as parameter for a callback. 

### ISSUE THREAD

**Issue by bit-garden** [TNorbury/command_palette#32]
This may be a bit silly since you can prompt a popup or something for it, but maybe have a prompt that can accept text as an argument for a callback. 

Something akin to this: 
```
/// Maybe return a new Command Palette Action for a chain of prompts?
CommandPaletteAction? someFun(String args) { ... }

CommandPaletteAction.nested(
  label: 'Manage Users', childrenActions: [
    CommandPaletteAction.WithCallback(
      label: 'Create', callback: someFun)
  ]
)
```

**Comment by TNorbury** [comment]
Implemented with #35

**Comment by TNorbury** [comment]
Implemented with #35
